resolver.go

```diff
@@ -123,7 +123,12 @@ func parseOSReleaseFile(s string, os *OSVersion) error {
 		}
 	}
 
-	if os.ID == "" || os.Version == "" {
+	versionNeeded := true
+	if os.IDLike == "arch" {
+		versionNeeded = false
+	}
+
+	if os.ID == "" || (versionNeeded && os.Version == "") {
```

Comment:
I think a more future proof solution would be to change the switch-case thing above to set the version to something like `0.0.0` in case there's something like `version.NewVersion(host.os.Version)`. 

```go
case "ID_LIKE":
		os.IDLike = unquote(fields[1])
		if os.IDLike == "arch" {
				os.Version = "0.0.0"
		}
```